li-apache-kafka-clients/src/main/java/com/linkedin/kafka/clients/producer/LiKafkaProducerImpl.java

```diff
@@ -128,6 +133,8 @@
   private final AtomicInteger _numThreadsInSend;
   private volatile boolean _closed;
 
+  private final Method _producerSupportsBoundedFlush;
```

Comment:
Can this be _boundedFlushMethod with a comment /* this is null if the producer does not have a method for time bounded flush. */

Reply:
Sure. Makes sense.